tests/test_list.py

```diff
@@ -435,7 +478,6 @@ def expected_windows_desktop_plus_wasm_5140(is_wasm_threaded: bool) -> Dict:
         ("--modules 5.14.0 win64_msvc2017_64", False, ["modules_by_arch", "win64_msvc2017_64"]),
         ("--modules 6.5.0 wasm_singlethread", True, ["modules_by_arch", "wasm_singlethread"]),
         ("--modules 6.5.0 wasm_multithread", True, ["modules_by_arch", "wasm_multithread"]),
-        ("--arch latest", True, ["architectures"]),
```

Comment:
This used to work, but now it refers to 6.8.1, where there are less archs available when going to all_os wasm: only `wasm_singlethread` `wasm_multithread`, and this was still excepting the other standard archs. Hence the removal. Tell me if I missed something and that it should be kept